tooling/bundler/src/bundle/windows/nsis.rs

```diff
@@ -56,6 +60,13 @@ const NSIS_REQUIRED_FILES: &[&str] = &[
   "Include/FileFunc.nsh",
   "Include/x64.nsh",
 ];
+#[cfg(not(target_os = "windows"))]
+const NSIS_REQUIRED_FILES: &[&str] = &[
+  "Plugins/x86-unicode/NScurl.dll",
+  "Plugins/x86-unicode/ApplicationID.dll",
+  "Plugins/x86-unicode/nsProcess.dll",
+  "Plugins/x86-unicode/nsis_semvercompare.dll",
+];
```

Comment:
you should also check for other important plugins, like `Mui2`, `FileFunc` and `x64`

Reply:
Hmm okay. What should we do if these files are missing? panic and ask the user to re-install nsis? or should we download the windows portable zip and extract them from there?

Reply:
panic and require a reinstall

Reply:
we have a classic Linux problem here. On most distros these plugins _should_ be in `/usr/share/nsis/Include` but we probably can't hardcode that since there are always distros that like to divert from the standards 🙄 

i'm gonna think about it a bit over the weekend because the idea of scanning the output of makensis for `!include: could not find: "MUI2.nsh"` etc doesn't sound nice either 😂

Reply:
I guess it is fine then, this is a rare case anyways and `--verbose` logs will show the error if someone reports it.